torch/testing/_internal/distributed/distributed_test.py

```diff
@@ -1007,12 +1007,12 @@ def test_periodic_model_averager(self):
             param = next(model.parameters())
             tensor = torch.ones_like(param.data) * rank
             expected_avg_tensor = torch.ones_like(param.data) * sum(range(world_size)) / world_size
-            averager = averagers.PeriodicModelAverager(model, warmup_steps=10, period=4)
+            averager = averagers.PeriodicModelAverager(model, warmup_steps=12, period=4)
```

Comment:
Shouldn't we have a test that combines PeriodicModelAverager with the local sgd hook?

Reply:
Hmm, I prefer keeping the unit test self-contained here.

Let me add another integration test called `test_post_localSGD_periodic_averaging` that involves both the comm hook and the model averaging component in the next PR.